intake_cesm/manage_collections.py

```diff
@@ -73,11 +78,22 @@ def _list_files_hsi(self):
 
         return filelist
 
+    def _list_files_input_file(self):
+        with open(self.urlpath, "r") as fid:
+            return fid.read().splitlines()
+
 
 class CESMCollections(object):
```

Comment:
Docstrings for the class constructor and some public methods would be useful to have.

Reply:
fair enough. I added some minimal text, but will need to go back thru carefully.  Still sketching out functionality...and working on this is a guilty pleasure (I should really be doing about 100 other things.

Reply:
I appreciate your work! Don't worry about it. I will take care of it in a separate PR that way you have time to do other things as well.